src/index.ts

```diff
@@ -72,6 +73,43 @@ export = (app: Probot) => {
     };
     await k8sApi.createNamespacedSecret(namespace,secret);
   });
+
+  //handle push event
+  app.on("push", async (context) => {
+
+    const configExists = Boolean(context.payload.commits?.reduce((acc, commit) => acc = [...acc, ...commit.added, ...commit.modified,...commit.removed], []).find((name:String) => name == "peribolos.yaml"));
```

Comment:
While this seems to work, I don't think it's correct. Reduce accepts a callback consuming an accumulator and current value. This callback has a body and it's return value is used as accumulator. Instead of a function body you're doing an assignment.. and since your assignment is not a statement but an expression, it also returns the assigned value:

```
a = 5
// returns 5
// vs.
var a = 5
// returns undefined
```

Hence what you're doing here is a bit of an abuse and you should return new accumulator value only.

It should be:
```suggestion
    const configExists = Boolean(context.payload.commits?.reduce((acc, commit) => [...acc, ...commit.added, ...commit.modified,...commit.removed], []).find((name: String) => name == "peribolos.yaml"));
```